apps/web/tailwind.config.js

```diff
@@ -17,13 +17,36 @@ module.exports = {
         "purple-background": "#e1d8f3",
         white: "#FFFFFF",
         black: "#1A202C",
+        "dark-black": "#000000",
```

Comment:
please extend a different set of names so that we don't override existing behavior, otherwise we can't merge it in since its not backward compatible

Reply:
Makes sense, I think I can put everything under the accent object.